pom.xml

```diff
@@ -968,10 +930,6 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-dbcp2</artifactId>
           </exclusion>
-          <exclusion>
```

Comment:
Overall I am +1 on this effort given that I am not a Maven expert and me and few others have spent considerable time recently on dependency problems where dependency exclusion (both in the context of Jackson and Netty) was used to avoid the problems we were facing. We have

- Sometimes the build problem where OSS build passed but when the OSS artifacts were pulled into LinkedIn artifactory and then used to build our internal product, build failed with dependency conflicts errors -- IIRC this happened for both Netty and Jackson
- Sometimes build went clean all the way, but then during deployment / restart with the new build we ran into runtime issues (typically NoClassDefFound error).

So just wondering if removing this exclusions can potentially reintroduce the two kinds of problems ? Also, given what you mentioned about the nearest definition priority, I am confused about why these problems happened in the first place ?

cc @jackjlli @jasperjiaguo who have also fixed dependency related problem